app/lib/helpers/directory_contents_helper.rb

```diff
@@ -0,0 +1,12 @@
+module Helpers
+
+  class DirectoryContentsHelper
+
+    def self.get_readable_file_names(directory:, filetype:)
+      Dir.chdir (Rails.root.to_s + directory)
```

Comment:
I'd avoid using `Dir.chdir` here.  I imagine it might screw with Rails as it's changing the working directory for the entire Ruby process.

You should be able to do everything you need to do without changing the working directory.